app/src/main/kotlin/com/wire/android/ui/home/conversations/MessageItem.kt

```diff
@@ -106,9 +112,12 @@ fun MessageItem(
             )
         }
 
+        // in case the message is marked as deleted and is expirable, it means that the message
+        // is belonging to the sender and we are waiting for the receiver timer to expire to permanently delete it
+        // we also do not want any background color
```

Comment:
I find this text a bit confusing. Would the following be slightly more clear?:
```suggestion
        // If a message is an expiring one and not deleted yet, it means that the current user is the receiver of it.
        // Once the receiver timer finishes, the message will be deleted locally, and send a signal to the sender so
        // he deletes the message locally as well. In this case, we don't want any background color either.
```

Reply:
this is not really true 🤔 I moved this comment and adjusted it a bit to line 182, pls take a look it it makes more sense to you now

Reply:
changed and resolved on the call